### PR TITLE
Merge latest change from main to custom tag feature branch

### PR DESCRIPTION
**Pull request by pengchen0692**
## Description
Merge latest change from main to custom tag feature branch

## Related issues
Addresses [AB#79464](https://microsofthealth.visualstudio.com/f8da5110-49b1-4e9f-9022-2f58b6124ff9/_workitems/edit/79464)

### CODE REVIEW

File: src/Microsoft.Health.Dicom.Core/DicomCoreResource.resx [hunk 1]
```diff
@@ -299,6 +299,16 @@ The first part date {1} should be lesser than or equal to the second part date {
     <value>The CR code '{0}' is not supported.</value>
     <comment>{0} VR Code.</comment>
   </data>
+  <data name="Forbidden" xml:space="preserve">
+    <value>Authorization failed.</value>
+  </data>
+  <data name="ErrorValidatingRoles" xml:space="preserve">
+    <value>Error validating roles:
+{0}</value>
```
Comment (wsugarman) [inline src/Microsoft.Health.Dicom.Core/DicomCoreResource.resx:307]: Just confirming: is the newline intended?

**Review by wsugarman — APPROVED** [review]

File: src/Microsoft.Health.Dicom.Core/DicomCoreResource.resx (hunk 1 above)
Comment (pengchen0692) [reply src/Microsoft.Health.Dicom.Core/DicomCoreResource.resx:307]: That's wired, let me double check.

Comment (pengchen0692) [reply src/Microsoft.Health.Dicom.Core/DicomCoreResource.resx:307]: yes... and the code in main is same, let me check with Brandon.
https://github.com/microsoft/dicom-server/blob/401bcc5ff16fbeb835b7ddddff1070c4243c9f78/src/Microsoft.Health.Dicom.Core/DicomCoreResource.resx#L309

Comment (pengchen0692) [reply src/Microsoft.Health.Dicom.Core/DicomCoreResource.resx:307]: it turns out coming from https://github.com/microsoft/fhir-server/blob/5f1b6365ef3484bed48581adeba3878309136381/src/Microsoft.Health.Fhir.Core/Resources.resx#L195

**Pull request merged**